adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -377,6 +378,21 @@ func CreateLuaCluster(interceptorCerts map[string][]byte, endpoint model.Interce
 	return processEndpoints(endpoint.ClusterName, &endpoint.EndpointCluster, interceptorCerts, endpoint.ClusterTimeout, endpoint.EndpointCluster.Endpoints[0].Basepath)
 }
 
+// CreateRateLimitCluster creates cluster relevant to the rate limit service
+func CreateRateLimitCluster() (*clusterv3.Cluster, []*corev3.Address, error) {
+	config, _ := config.ReadConfigs()
+	rlCluster := &model.EndpointCluster{
+		Endpoints: []model.Endpoint{
+			{
+				Host:    config.Envoy.RateLimit.Hostname,
+				URLType: httpsURLType,
+				Port:    config.Envoy.RateLimit.Port,
+			},
+		},
+	}
+	return processEndpoints(rateLimitClusterName, rlCluster, nil, 20, "")
```

Comment:
Shall we assign the return cluster to a variable and then do the modifications specific to rate limit cluster? Then we can avoid having cluster specific stuff in the generic cluster creation method.